src/openzaak/conf/includes/base.py

```diff
@@ -667,6 +688,12 @@
         "elasticapm.contrib.django",
     ]
 
+
+#
+# LOG OUTGOING REQUESTS
+#
+LOG_OUTGOING_REQUESTS_DB_SAVE = config("LOG_OUTGOING_REQUESTS_DB_SAVE", default=True)
```

Comment:
Let's make default=False for backwards compatibility.

Reply:
Done